cli/src/commands/git/clone.rs

```diff
@@ -47,7 +49,7 @@ use crate::ui::Ui;
 #[derive(clap::Args, Clone, Debug)]
 pub struct GitCloneArgs {
     /// URL or path of the Git repo to clone
-    #[arg(value_hint = clap::ValueHint::DirPath)]
+    #[arg(add = ArgValueCompleter::new(complete::new_git_remote_url))]
```

Comment:
This is a little unfortunate. I tried to keep both, but that wouldn't work, the directory path value hint would just be ingored.

This means that file paths are not completed anymore with this change, a slight regression. The only way I see to fix this is to reimplement the logic in our own value completer. (or maybe ask clap to enable this use case)